ethereum/src/libraries/Environment.sol

```diff
@@ -1,5 +1,6 @@
 pragma solidity ^0.8.24;
 
+/*
```

Comment:
Remove these addresses that you've commented out

Reply:
done